tests/modules/services/window-managers/sway/sway-followmouse-legacy.nix

```diff
@@ -0,0 +1,35 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    wayland.windowManager.sway = {
+      enable = true;
+
+      config = {
+        focus.followMouse = false;
+        menu = "${pkgs.dmenu}/bin/dmenu_run";
+        terminal = "${pkgs.termite}/bin/termite";
+        bars = [{ command = "${pkgs.swaybar}/bin/swaybar"; }];
```

Comment:
This could cause problems with #1386. I think it would be preferable if these tests only test followmouse behaviour.

Reply:
The output is non-deterministic (aka untestable) without this. We can hold on this and I can rebase once #1386 is fixed, I don't mind.

Reply:
I'm not sure how this causes non-determinism but maybe `bars = [ ]` would be cleaner

Reply:
By default, currently pre-1386-fix, it includes a reference to the swaybar binary. The swaybar binary path will differ as `pkgs` move and thus the path that is output in the exec line in the tested sway config changes (thus becoming untestable). It's the reason for this entire path faking mechanism.

`bars = []` would probably work too though... let me try.

Reply:
Cool, thanks for the suggestion, this was a nice simplification to make.

Reply:
Are these problems not solved by the overrides below?

Reply:
I haven't actually tried so this could be wrong but if you override `rxvt-unicode` `dmenu` and `sway` then I don't think any store paths should end up in a config file that only has `focus.followMouse = false`

Reply:
So... dug into this, it doesn't work because the swaybar line refers to `config.package` which defaults to `defaultSwayPackage` which is an override of `pkgs.sway`, so no, the override mechanism doesn't work.

Fortunately, your suggestion worked and removed the problem entirely.

Reply:
Ah this makes sense, sorry for being fussy. It still seems like one of overriding `termite` and `rxvt-unicode` is unnecessary to me.

Reply:
No, you're right, I'm doing that now. Menu is still necessary, the default includes `... | findutils/../xargs`.

Reply:
Okay, marking resolved again. This is the most minimal I can make it. I think I touched on why sway/dmenu overrides are still necessary. lmk.